fe/fe-core/pom.xml

```diff
@@ -65,6 +73,13 @@ under the License.
                 <starrocks.thrift>${env.STARROCKS_THIRDPARTY}/installed/bin/thrift</starrocks.thrift>
             </properties>
         </profile>
+        <profile>
+            <id>windows</id>
```

Comment:
Why don't this property have activation

Reply:
we should active this profile manually.
eg:
![image](https://user-images.githubusercontent.com/34912776/178114492-54d63fea-9f20-4f68-b897-4160d466146f.png)